sdk/go/common/workspace/plugins.go

```diff
@@ -747,14 +747,18 @@ func buildHTTPRequest(pluginEndpoint string, token string) (*http.Request, error
 
 func getHTTPResponse(req *http.Request) (io.ReadCloser, int64, error) {
 	logging.V(9).Infof("full plugin download url: %s", req.URL)
-	logging.V(9).Infof("plugin install request headers: %v", req.Header)
+	// This logs at level 11 because it could include authentication headers, we reserve log level 11 for
```

Comment:
Docs? Would love to see what else is in it.

Reply:
We don't really have docs for this, but the sass api client also logs headers at level 11.

Reply:
I'll update that